docs/appendices/release-notes/5.7.0.rst

```diff
@@ -108,10 +113,14 @@ Performance and Resilience Improvements
 Administration and Operations
 -----------------------------
 
-- Added the ability to set a
+- `meteoiker <https://github.com/meteoiker>`_ added the ability to set a
   :ref:`use_path_style_access <sql-create-repo-s3-use_path_style_access>` for
```

Comment:
Hi @meteoiker, I haven't found full name in your account, so using GH handle.
Feel free to suggest a version with a full name if you want.